checks/code_review.go

```diff
@@ -118,14 +118,15 @@ func githubCodeReview(c *checker.CheckRequest) (int, string, error) {
 		// Check if the PR is committed by someone other than author. this is kind
 		// of equivalent to a review and is done several times on small prs to save
 		// time on clicking the approve button.
-		if !foundApprovedReview {
-			if !pr.MergeCommit.AuthoredByCommitter {
-				c.Dlogger.Debug3(&checker.LogMessage{
-					Text: fmt.Sprintf("found pr with committer different than author: %d", pr.Number),
-				})
-				totalReviewed++
-			}
+		if !foundApprovedReview &&
+			pr.MergeCommit.Committer.Login != pr.Author.Login {
+			c.Dlogger.Debug3(&checker.LogMessage{
+				Text: fmt.Sprintf("found PR#%d with committer different than author: %s != %s",
```

Comment:
Nit, change log text to: `committer (%s) different from author (%s)` to improve clarity.